.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:10.19
+    - image: circleci/node:12.21
```

Comment:
Bumped the version of node on circle from 10 to 12. 12 is already in maintenance mode but I want to be somewhat conservative.

Reply:
At the bottom of package.json there's a node version constraint. Do we want to update that one too?

https://github.com/intercom/contentful-typescript-codegen/blob/7de8bd824628920b35db261ce3b535cf4e0c5847/package.json#L74-L76